mm2src/mm2_main/src/lp_swap/taker_swap.rs

```diff
@@ -1824,6 +1824,30 @@ impl TakerSwap {
 
         let tx_hash = transaction.tx_hash_as_bytes();
         info!("Maker payment spend tx {:02x}", tx_hash);
+
+        // we should wait for only one confirmation to make sure spend maker payment transaction is not failed
+        let confirm_maker_payment_spend_input = ConfirmPaymentInput {
+            payment_tx: transaction.tx_hex(),
+            confirmations: std::cmp::min(1, self.r().data.maker_payment_confirmations),
```

Comment:
Is there any reason to involve 'maker_payment_confirmations' if we always want exactly 1 conf?
(And what would be if maker_payment_confirmations set to 0? Like if someone wants to run the whole swap with no confs at all?)